tests/ignored-metric-list

```diff
@@ -0,0 +1,79 @@
+authenticated_user_requests
```

Comment:
You may create a sub-folder `tests/pkg/testdata` to store this ignore-metrics-list.